package.json

```diff
@@ -224,6 +224,7 @@
     "typed-redux-saga": "^1.4.0",
     "typesafe-actions": "4.4.2",
     "unist-util-visit": "^4.1.0",
+    "url": "^0.11.4",
```

Comment:
Why this package version is changed?

Reply:
I was having some troubles with my environment for some reasons and I tried to update both the Pods and some package in the troubleshooting phase. That's a residue that I can remove